canonical-web-and-design/ubuntu.com
# Update AI in the developer menu + remove /kubeflow pages

## Done

- Update AI in developer menu
- Update AI in the enterpris menu
- Add AI and remove kubeflow from the footer
- Update links around the site from '/kubeflow' to '/ai' pages including the sitemap and HACKING.md to prevent redirects

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare the menu to the [copy doc](https://docs.google.com/document/d/1JzZPBydcdyV96Hu4xRpa7VDU_WREr6gb05bL7GlQaYY/edit?ts=602f8b1e#)


## Issue / Card

Fixes [#3836](https://github.com/canonical-web-and-design/web-squad/issues/3836)
